packages/ppx_protocol_conv_yaml/ppx_protocol_conv_yaml.5.0.0/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "Anders Fugmann <anders@fugmann.net>"
+authors: "Anders Fugmann"
+homepage: "https://github.com/andersfugmann/ppx_protocol_conv"
+dev-repo: "git+https://github.com/andersfugmann/ppx_protocol_conv"
+bug-reports: "https://github.com/andersfugmann/ppx_protocol_conv/issues"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "ppx_protocol_conv" {= "5.0.0"}
+  "yaml"
+  "yaml" {with-test & >= "2.0.0"}
```

Comment:
yaml is present twice. Which one is true?

Reply:
Nice catch, Thanks.